app/controllers/ConfirmationPageController.scala

```diff
@@ -42,13 +43,14 @@ class ConfirmationPageController @Inject()(
 
   def onPageLoad(fileRole: FileRole): Action[AnyContent] = (identify andThen getData andThen requireData).async {
     implicit request =>
+      val dates = new CheckYourAnswersHelper(request.userAnswers)
       for {
         email <- customsDataStoreConnector.getEmail(request.eori).map {
           case Right(email) => Some(email)
           case Left(_) => None
         }.recover { case _ => None }
         _ <- sessionRepository.clear(request.internalId)
         returnLink = appConfig.returnLink(fileRole, request.userAnswers)
-      } yield Ok(view(email, fileRole, returnLink))
+      } yield Ok(view(email, fileRole, returnLink, dates))
```

Comment:
Better to  pass only the date range string (instead of passing CheckYourAnswersHelper object) using as below?
Or is there any specific reason to pass the entire CheckYourAnswersHelper object? Its always good to keep the view as light as possible.

dates.dateRows(fileRole)